website/content/docs/concepts/domain-model/targets.mdx

```diff
@@ -51,63 +51,6 @@ TCP targets have the following additional attributes:
 - `default_port` - (optional)
   The default port to set on this target.
 
-- `egress_worker_filter` - (optional)
-  A boolean expression to [filter][] which egress workers can handle sessions
-  for this target.
-  Egress worker filters determine which workers are used to access targets.
-  You can configure an egress filter to enable [multi-hop](/boundary/docs/configuration/worker/pki-worker#multi-hop-workershcp-only) connections.
-  If you do not configure an egress filter, then Boundary uses a single worker to connect to the controller.
-
-- `ingress_worker_filter` - (optional) <sup>HCP Only</sup>
-  A boolean expression to [filter][] which ingress workers can handle sessions
-  for this target.
-  Ingress worker filters determine which workers you connect with to initiate a session.
-  If you do not configure an ingress filter, Boundary selects a front line worker for the session.
-  A front line worker is any worker directly connected to the control plane; for HCP Boundary this will be an HCP worker.
-
-- `session_connection_limit` - (required)
-  The cumulative number of TCP connections allowed during a session.
-  A -1 value means no limit.
-  The default is -1.
-  The value must be greater than 0 or exactly -1.
-
-- `session_max_seconds` - (required)
-  The maximum duration of an individual session between the user and the target.
-  All connections for a session are closed
-  and the session is terminated
-  when a session reaches the maximum duration.
-  The default is 8 hours (28800 seconds).
-  This value must be greater than 0.
```

Comment:
Egress worker filters are in OSS as well, using `egress_worker_filter` as a replacement to the original `worker_filter`, not previously documented on this page. It should just be `ingress_worker_filter` that is HCP-only

```
- `egress_worker_filter` - (optional)
  A boolean expression to [filter][] which egress workers can handle sessions
  for this target. Replaces `worker_filter` used prior to Boundary 0.12.0.
  Egress worker filters determine which workers are used to access targets.
```

Reply:
this can include the original multi-hop content after next week